lib/vquic/ngtcp2.c

```diff
@@ -1600,4 +1731,20 @@ CURLcode Curl_quic_done_sending(struct connectdata *conn)
 
   return CURLE_OK;
 }
+
+/*
+ * Called from http.c:Curl_http_done when a request completes.
+ */
+void Curl_quic_done(struct connectdata *conn, bool premature)
+{
+  (void)premature;
+  if(conn->handler == &Curl_handler_http3) {
+    /* only for HTTP/3 transfers */
+    struct HTTP *stream = conn->data->req.protop;
+    Curl_safefree(stream->overflow_buf);
+  }
+
+  return CURLE_OK;
```

Comment:
~~~
vquic/ngtcp2.c:1747:10: error: ‘return’ with a value, in function returning void [-Werror]
   return CURLE_OK;
          ^~~~~~~~

Reply:
Thanks, will make sure to build this with different compilers before the next commit.